iTunesSearchHW23/iTunesSearch/Common/ItemDisplaying.swift

```diff
@@ -1,12 +1,14 @@
-
+// MARK: - Importing framewroks
 import UIKit
 
+// MARK: - Protocols
 protocol ItemDisplaying {
     var itemImageView: UIImageView! { get set }
     var titleLabel: UILabel! { get set }
     var detailLabel: UILabel! { get set }
 }
 
+// MARK: - Extensions
```

Comment:
If u add such mark, it should contain name of object or short description of functionality